crypto/fipsmodule/dh/check.c

```diff
@@ -125,6 +126,13 @@ int DH_check(const DH *dh, int *out_flags) {
   BIGNUM *t1 = NULL, *t2 = NULL;
 
   *out_flags = 0;
+
+  /* Don't do any checks at all with an excessively large modulus */
```

Comment:
Tiny tiny nit for our coding style
```suggestion
  // Don't do any checks at all with an excessively large modulus
```